src/index.html

```diff
@@ -4,13 +4,40 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0,
+      maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="bigSearchForm"
+      action="/"
+      method="get"
+      data-qa="big"
+    >
+      <input
+        class="bigSearchInput input"
```

Comment:
use one of the inputs as the basic case, and override necessary styles for other cases. You can  add big input style to the basic input class and rewrite style for small input